app/containers/Scatter/saga.js

```diff
@@ -55,6 +56,31 @@ function* watchScatterConnect() {
   yield takeLatest(CONNECT_ACCOUNT, getEosAccount);
 }
 
+//
+// Refresh account data
+//
+
+function* refreshEosAccountData() {
+  const accountName = yield select(makeSelectEosAccount());
+  const eosClient = yield select(EosClient());
+  try {
+    if (accountName && accountName !== 'Attach an Account') {
```

Comment:
This looks very fragile. Isn't there a better way to make sure that the account is genuine?

Reply:
Currently its just a string. I'm probably doing a "bad thing" by setting it directly to attach an account... should just be setting it to null or undefined. This would need some refactoring though.

Reply:
Yes this should be set to `undefined`, it is more proper.

No problem, let's tackle that in a separate PR